app/views/document_images/_image_list.html.erb

```diff
@@ -15,9 +15,17 @@ end %>
     <% end %>
   <% end %>
 
-  <% delete_image_button = capture do %>
-    <%= form_tag delete_document_lead_image_path(@document, image), method: :delete, class: "app-inline-block" do %>
-      <button class="govuk-link app-link--destructive">Delete image</button>
+  <% if image.id == @document.lead_image_id %>
```

Comment:
The way this is presented is a little bit confusing with the capture assigned to same variable name in each condition. 

I wonder if it'd be easier to follow if it was `<% delete_image_button = capture do %>` and then the conditional inside that?